robottelo/cli/hammer.py

```diff
@@ -38,11 +40,36 @@ def _csv_reader(output):
             yield row
 
 
+def _normalize(header):
+    """Replace empty spaces with '-' and lower all chars"""
+    return header.replace(' ', '-').lower()
+
+
+def parse_json(stdout):
+    """Parse JSON output from Hammer CLI and convert it to python dictionary
+    while normalizing keys."""
+    parsed = json.loads(stdout)
+    return _normalize_obj(parsed)
+
+
+def _normalize_obj(obj):
+    """Normalize all dict's keys replacing empty spaces with "-" and lowering
+    chars"""
```

Comment:
"""Normalize all dict's keys replacing empty spaces with "-" and lowering
chars
"""